pkg/web/src/app/routes.tsx

```diff
@@ -15,6 +15,9 @@ import { VMMigrationDetails } from '@app/Plans/components/VMMigrationDetails';
 import { LoginHandlerComponent } from './common/LoginHandlerComponent';
 import { RedirectToLogin } from './common/RedirectToLogin';
 import { NotFound } from './NotFound';
+import { IEnvVars } from './common/types';
+
+export const ENV: IEnvVars = window['_env'] || {};
```

Comment:
Oh, we also can replace this one with a `import { ENV } from '@app/common/constants';` as well